### PR TITLE
Document the --hypothesis-strategy cli param

### PR DESCRIPTION
**Pull request by adamdabbracci**
Signed-off-by: Adam D'Abbracci <adamdabbracci@adams-mbp.lan>

### CODE REVIEW

File: sources/reference/usage/run.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ To test that you have a valid experiment you can pass the `--dry` option.
 You can run an experiment and skip the experiment's validation using the 
 `--no-validation` option.
 
+## Run an experiment with different steady state strategies
+By default, the steady state will be tested before and after an experiment runs. However, you can specify a different strategy through the `--hypothesis-strategy` parmeter. The options are:
```
Comment (ciaranevans) [inline sources/reference/usage/run.md:40]:
```suggestion
By default, the steady state will be tested before and after an experiment runs. However, you can specify a different strategy through the `--hypothesis-strategy` parameter. The options are:
```

**Review by ciaranevans — CHANGES_REQUESTED** [review]
Thanks for the contribution @adamdabbracci!

Looks good to me apart from the small typo!

**Comment by ciaranevans** [comment]
@adamdabbracci 👋 Just a reminder that you need to DCO sign **every** commit you push! 

If you're a frequent contributor to our repos, what I've done is setup a commit template in my `chaostoolkit/` dir with:

```
Signed-off-by: Ciaran Evans <ciaran@reliably.com>
```

In a file called `chaostoolkit-commit-template.txt`

You can then set it to just the local dir by:

```
git config --local commit.template fully/qualified/path/to/template/file
```

**Comment by adamdabbracci** [comment]
Whoops! I committed your fix directly from the UI, so I completely forgot about the DCO. I've tried to sign it for that commit, but it doesn't seem to be picking up, so I must be missing something obvious.

Checkout the PR:
`gh pr checkout 106 -R chaostoolkit/chaostoolkit-documentation`

Sign off:
`git commit --amend --no-edit --signoff`

Push:
`git push --force-with-lease origin add-hypothesis-strategy`

**Comment by ciaranevans** [comment]
I wonder if it's something silly like we're both on the commit or something! I'll have a look!

**Comment by ciaranevans** [comment]
@adamdabbracci are you signing off with an email that is available on your GitHub Account?

Just looking at the UI above, the pushes are assigned to you, but the commits appear to be profile-less

Can you double check that your email is listed here https://github.com/settings/emails?

**Comment by adamdabbracci** [comment]
I'm going to close this PR and open a new one... with the signature correct from the beginning hopefully... thank you for trying to troubleshoot!

**Pull request closed without merging**